### DISCUSSION TITLE
Error " C:/Users/Miss Na/AppData/Local/qv2ray/generated/config.gen.json v2ray.com/core/infra/conf: failed to parse to outbound detour config. > v2ray.com/core/infra/conf: unknown config id: > exit status 4294967295 " #2953

### DISCUSSION THREAD

**Discussion by lamphuongvnn2** [v2fly/v2ray-core#1540] [Q&A]
I tried connecting on my phone with v2ray and it works fine.. But on my PC, I get this error (q2ray vs v2ray) what should I do? I try to import with Json from my phone.
**Link image Json:** https://snipboard.io/Zhaq49.jpg

V2Ray 4.31.0 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.15.2 windows/amd64)
A unified platform for anti-censorship.
main: failed to read config files: [C:/Users/Miss Na/AppData/Local/qv2ray/generated/config.gen.json] > v2ray.com/core/main/json: failed to execute v2ctl to convert config file. > v2ray.com/core/common/platform/ctlcmd: failed to execute v2ctl:
v2ctl> Read config: C:/Users/Miss Na/AppData/Local/qv2ray/generated/config.gen.json
v2ray.com/core/infra/conf: failed to parse to outbound detour config. > v2ray.com/core/infra/conf: unknown config id: > exit status 4294967295

=====
// Config file of V2Ray. This file follows standard JSON format, with comments support.
// Uncomment entries below to satisfy your needs. Also read our manual for more detail at
// https://www.v2ray.com/
{
"log": {
// By default, V2Ray writes access log to stdout.
// "access": "/path/to/access/log/file",

// By default, V2Ray write error log to stdout.
// "error": "/path/to/error/log/file",

// Log level, one of "debug", "info", "warning", "error", "none"
"loglevel": "warning"
},
// List of inbound proxy configurations.
"inbounds": [{
// Port to listen on. You may need root access if the value is less than 1024.
"port": 1080,

// IP address to listen on. Change to "0.0.0.0" to listen on all network interfaces.
"listen": "127.0.0.1",

// Tag of the inbound proxy. May be used for routing.
"tag": "socks-inbound",

// Protocol name of inbound proxy.
"protocol": "socks",

// Settings of the protocol. Varies based on protocol.
"settings": {
  "auth": "noauth",
  "udp": false,
  "ip": "127.0.0.1"
},

// Enable sniffing on TCP connection.
"sniffing": {
  "enabled": true,
  // Target domain will be overriden to the one carried by the connection, if the connection is HTTP or HTTPS.
  "destOverride": ["http", "tls"]
}
}],
// List of outbound proxy configurations.
"outbounds": [{
// Protocol name of the outbound proxy.
"protocol": "freedom",

// Settings of the protocol. Varies based on protocol.
"settings": {},

// Tag of the outbound. May be used for routing.
"tag": "direct"
},{
"protocol": "blackhole",
"settings": {},
"tag": "blocked"
}],

// Transport is for global transport settings. If you have multiple transports with same settings
// (say mKCP), you may put it here, instead of in each individual inbound/outbounds.
//"transport": {},

// Routing controls how traffic from inbounds are sent to outbounds.
"routing": {
"domainStrategy": "IPOnDemand",
"rules":[
{
// Blocks access to private IPs. Remove this if you want to access your router.
"type": "field",
"ip": ["geoip:private"],
"outboundTag": "blocked"
},
{
// Blocks major ads.
"type": "field",
"domain": ["geosite:category-ads"],
"outboundTag": "blocked"
}
]
},

// Dns settings for domain resolution.
"dns": {
// Static hosts, similar to hosts file.
"hosts": {
// Match v2ray.com to another domain on CloudFlare. This domain will be used when querying IPs for v2ray.com.
"domain:v2ray.com": "www.vicemc.net",

  // The following settings help to eliminate DNS poisoning in mainland China.
  // It is safe to comment these out if this is not the case for you.
  "domain:github.io": "pages.github.com",
  "domain:wikipedia.org": "www.wikimedia.org",
  "domain:shadowsocks.org": "electronicsrealm.com"
},
"servers": [
  "1.1.1.1",
  {
    "address": "114.114.114.114",
    "port": 53,
    // List of domains that use this DNS first.
    "domains": [
      "geosite:cn"
    ]
  },
  "8.8.8.8",
  "localhost"
]
},

// Policy controls some internal behavior of how V2Ray handles connections.
// It may be on connection level by user levels in 'levels', or global settings in 'system.'
"policy": {
// Connection policys by user levels
"levels": {
"0": {
"uplinkOnly": 0,
"downlinkOnly": 0
}
},
"system": {
"statsInboundUplink": false,
"statsInboundDownlink": false,
"statsOutboundUplink": false,
"statsOutboundDownlink": false
}
},

// Stats enables internal stats counter.
// This setting can be used together with Policy and Api.
//"stats":{},

// Api enables gRPC APIs for external programs to communicate with V2Ray instance.
//"api": {
//"tag": "api",
//"services": [
// "HandlerService",
// "LoggerService",
// "StatsService"
//]
//},

// You may add other entries to the configuration, but they will not be recognized by V2Ray.
"other": {}
}

**Comment by lamphuongvnn2** [comment]
Groups is work ?

**Comment by kallydev** [comment]
v2fly-core is not support the configuration format shown in the screenshot, you can solve this problem by writing your own configuration file or using some conversion tools.

I see you are using qv2ray, this client provided a graphical configuration editor, you just need to fill in all the parameters correctly.

https://qv2ray.net
https://www.v2fly.org

**Comment by lamphuongvnn2** [comment]
Link image: https://snipboard.io/FELnuP.jpg

I try to follow the configuration guide, but it doesn't work.. Can you tell me where I have misconfigured?

**Comment by kallydev** [comment]
> Link image: https://snipboard.io/FELnuP.jpg
> 
> 
> 
> I try to follow the configuration guide, but it doesn't work.. Can you tell me where I have misconfigured?

You've filled in the wrong encryption method and the extra TLS configuration. I recall that this configuration format belong to Shadowrocket, which also provided a `Copy URL` option below the `Copy JSON` option, so you can import nodes in qv2ray by copying and pasting the URL.

The URL format start with `vmess://`.
